spec/docs/swagger_spec.rb

```diff
@@ -10,10 +10,57 @@
 
   context 'has valid paths' do
     it do
+      expect(subject).to validate(:get, '/manifests/{id}', 200, params)
+    end
+    it do
+      expect(subject).to validate(:get, '/manifests/{id}', 404, unknown_manifest_params)
+    end
+    it do
+      expect(subject).to validate(
+        # TODO must we require 'manifest' key? that is inconsistent with PATCH and /validate
```

Comment:
We could do that. 

I've been moving our API over to use params because then the API controllers and web controllers are working more similarly. I like the pattern of having shared controller concerns for the API and Web params.

Do you think the benefit of having the API controllers work differently is worth the risk of having to update two controllers in the case that, say, we add a field to the JSON schema?